.github/workflows/specimin_evaluation_CI.yml

```diff
@@ -0,0 +1,132 @@
+name: specimin_evaluation_CI
```

Comment:
Can we change the name of this job to be more descriptive of what it actually does? Maybe something like "check_compilable_percentage"?

There is already another CI job called "specimin-evaluation", so I'm trying to avoid confusion.